aries-site/src/data/structure.js

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable max-len */
+import {
+  IconComponents,
+  IconDesign,
+  IconDevelop,
+  IconFoundation,
+  IconGuidelines,
+  IconResources,
+  IconBrand,
+  IconColor,
+  IconIcons,
+  IconLayout,
+  IconType,
+  IconToken,
+} from '../components/icons';
+
+export const structure = [
+  {
```

Comment:
To be comprehensive, we should add an object for the Home page.
```
{
  name: 'Home',
  pages: ['Guidelines', ... ],
}
```

Would require modification to this filter mentioned here: https://github.com/hpe-design/aries/pull/168/files?file-filters%5B%5D=.js&hide-deleted-files=true#r363999070